### PR TITLE
working on ticket 3

### PR DESCRIPTION
**Pull request by DimitrinaT**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test.only("test endpoint replies with status 200 and returns an array of topic objects ", () => {
```
Comment (EddTheDuck) [inline __tests__/app.test.js:16]: Please remove any .only so the whole test suite runs before submitting

**Review by EddTheDuck — CHANGES_REQUESTED** [review]
resubmit without .only

File: __tests__/app.test.js (hunk 1 above)
Comment (DimitrinaT) [reply __tests__/app.test.js:16]: should i resend it

**Review by EddTheDuck — APPROVED** [review]
Looks good!

**Pull request merged**